packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -433,20 +450,21 @@ export class PerformanceEvent {
     }
 
     public cancel(props?: ITelemetryProperties, error?: any): void {
-        this.reportEvent("cancel", props, error);
+        if (this.markers.cancel !== undefined) {
```

Comment:
this seems wrong, as it requires markers.cancel be set to log the cancel event at all. this will make all existing calls not log. i think markers should be called skip markers, and list which markers to skip, not those to include.